Unable to build on Windows

* Platform: Windows 10

#### Steps to reproduce:

* `run yarn / yarn install`

#### Notes:
* I have Python 2 and MSBuild installed
* Running `npm install` goes through succesfully, but then blows up on `npm start`

#### Error:

```
yarn install v1.1.0
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.0.14: The platform "win32" is incompatible with this module.
info "fsevents@1.0.14" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning "ajv-keywords@1.1.1" has unmet peer dependency "ajv@>=4.2.0".
warning "pegjs-loader@0.5.1" has incorrect peer dependency "pegjs@^0.10.0".
warning "serviceworker-loader@0.1.0" has incorrect peer dependency "webpack@>=0.9 <2".
[4/4] Building fresh packages...
[-/2] ⡀ waiting...
[2/2] ⡀ node-sass:   Building solution configuration "Release|x64".
[-/2] ⢀ waiting...
[-/2] ⢀ waiting...
error C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass: Command failed.
Exit code: 1
Command: node scripts/build.js
Arguments:
Directory: C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass
Output:
Building: C:\Program Files (x86)\nodejs\node.exe C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-gyp\bin\node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=
gyp info it worked if it ends with ok
gyp verb cli [ 'C:\\Program Files (x86)\\nodejs\\node.exe',
gyp verb cli   'C:\\Users\\Gigo\\Documents\\GitHub\\bemuse\\node_modules\\node-gyp\\bin\\node-gyp.js',
gyp verb cli   'rebuild',
gyp verb cli   '--verbose',
gyp verb cli   '--libsass_ext=',
gyp verb cli   '--libsass_cflags=',
gyp verb cli   '--libsass_ldflags=',
gyp verb cli   '--libsass_library=' ]
gyp info using node-gyp@3.4.0
gyp info using node@8.6.0 | win32 | x64
gyp verb command rebuild []
gyp verb command clean []
gyp verb clean removing "build" directory
gyp verb command configure []
gyp verb check python checking for Python executable "python2" in the PATH
gyp verb `which` succeeded python2 C:\Python27\python2.EXE
gyp verb check python version `C:\Python27\python2.EXE -c "import platform; print(platform.python_version());"` returned: "2.7.13\r\n"
gyp verb get node dir no --target version specified, falling back to host node version: 8.6.0
gyp verb command install [ '8.6.0' ]
gyp verb install input version string "8.6.0"
gyp verb install installing version: 8.6.0
gyp verb install --ensure was passed, so won't reinstall if already installed
gyp verb install version is already installed, need to check "installVersion"
gyp verb got "installVersion" 9
gyp verb needs "installVersion" 9
gyp verb install version is good
gyp verb get node dir target node version installed: 8.6.0
gyp verb build dir attempting to create "build" dir: C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build
gyp verb build dir "build" dir needed to be created? C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build
gyp verb build/config.gypi creating config file
gyp verb build/config.gypi writing out config file: C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\config.gypi
gyp verb config.gypi checking for gypi file: C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\config.gypi
gyp verb common.gypi checking for gypi file: C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\common.gypi
gyp verb gyp gyp format was not specified; forcing "msvs"
gyp info spawn C:\Python27\python2.EXE
gyp info spawn args [ 'C:\\Users\\Gigo\\Documents\\GitHub\\bemuse\\node_modules\\node-gyp\\gyp\\gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'msvs',
gyp info spawn args   '-G',
gyp info spawn args   'msvs_version=auto',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Users\\Gigo\\Documents\\GitHub\\bemuse\\node_modules\\node-sass\\build\\config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Users\\Gigo\\Documents\\GitHub\\bemuse\\node_modules\\node-gyp\\addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Users\\Gigo\\.node-gyp\\8.6.0\\include\\node\\common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=C:\\Users\\Gigo\\.node-gyp\\8.6.0',
gyp info spawn args   '-Dnode_gyp_dir=C:\\Users\\Gigo\\Documents\\GitHub\\bemuse\\node_modules\\node-gyp',
gyp info spawn args   '-Dnode_lib_file=node.lib',
gyp info spawn args   '-Dmodule_root_dir=C:\\Users\\Gigo\\Documents\\GitHub\\bemuse\\node_modules\\node-sass',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'C:\\Users\\Gigo\\Documents\\GitHub\\bemuse\\node_modules\\node-sass\\build',
gyp info spawn args   '-Goutput_dir=.' ]
gyp verb command build []
gyp verb build type Release
gyp verb architecture x64
gyp verb node dev dir C:\Users\Gigo\.node-gyp\8.6.0
gyp verb found first Solution file build/binding.sln
gyp verb could not find "msbuild.exe" in PATH - finding location in registry
gyp verb "Release" dir needed to be created? null
gyp verb copying "node.lib" for x64 C:\Users\Gigo\.node-gyp\8.6.0\Release\node.lib
gyp info spawn C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe
gyp info spawn args [ 'build/binding.sln',
gyp info spawn args   '/nologo',
gyp info spawn args   '/p:Configuration=Release;Platform=x64' ]
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
Build started 10/9/2017 21:38:35.
Project "C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\binding.sln" on node 1 (default targets).
ValidateSolutionConfiguration:
  Building solution configuration "Release|x64".
Project "C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\binding.sln" (1) is building "C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\binding.vcxproj.metaproj" (2) on node 1 (default targets).
Project "C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\binding.vcxproj.metaproj" (2) is building "C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\src\libsass.vcxproj" (3) on node 1 (default targets).
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Platforms\x64\PlatformToolsets\v140\Toolset.targets(36,5): error MSB8036: The Windows SDK version 8.1 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution". [C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\src\libsass.vcxproj]
Done Building Project "C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\src\libsass.vcxproj" (default targets) -- FAILED.
Done Building Project "C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\binding.vcxproj.metaproj" (default targets) -- FAILED.
Done Building Project "C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\binding.sln" (default targets) -- FAILED.

Build FAILED.

"C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\binding.sln" (default target) (1) ->
"C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\binding.vcxproj.metaproj" (default target) (2) ->
"C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\src\libsass.vcxproj" (default target) (3) ->
(Desktop_PlatformPrepareForBuild target) ->
  C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Platforms\x64\PlatformToolsets\v140\Toolset.targets(36,5): error MSB8036: The Windows SDK version 8.1 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution". [C:\Users\Gigo\Documents\GitHub\bemuse\node_modules\node-sass\build\src\libsass.vcxproj]

    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.12
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1
```

--- a/package.json
+++ b/package.json
@@ -94,7 +94,7 @@
     "mocha": "^2.1.0",
     "node-env": "^0.1.6",
     "node-notifier": "^4.0.3",
-    "node-sass": "^4.5.0",
+    "node-sass": "^4.5.3",
     "pegjs": "^0.9.0",
     "pegjs-loader": "^0.5.1",
     "react-hot-loader": "^3.0.0-beta.6",

